composer.json

```diff
@@ -16,11 +16,11 @@
     }
   ],
   "require": {
-    "php": "~7.1.3||~7.2.0||~7.3.0",
+    "php": "^7.1",
```

Comment:
Is this right?

Reply:
That line change is from an older commit, "php": "^7.1", is in master branch